transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -228,6 +230,7 @@ private void registerService(BundleContext bundleContext) {
             mimeTypes.add(mimeType);
         }
         mimeTypes.add("image/jp2");
+        mimeTypes.add("image/bmp");
```

Comment:
You didn't need to register a mimetype for tiff? Odd that the mimetype would already be advertised without a service provider to back it.